pypolygames/model_zoo/amazons_model.py

```diff
@@ -120,16 +120,16 @@ def _forward(self, x: torch.Tensor, return_logit: bool):
             pi_logit[:, c_prime + h_prime :].reshape(-1, 1, 1, w_prime), 3
         )
         pi = v1 * v2 * v3
-        # pi = nn.functional.softmax(pi.view(pi.shape[0], -1), 1).reshape(pi.shape)
+        pi = nn.functional.softmax(pi.view(pi.shape[0], -1), 1).reshape(pi.shape)
         # This representation is not sparse, that's a temporary hack
         # for testing the idea of a cartesian product.
         return v, pi
 
     @torch.jit.script_method
     def forward(self, x: torch.Tensor):
-        v, pi_logit = self._forward(x, True)
-        pi_logit = pi_logit.view(-1, self.c_prime, self.h_prime, self.w_prime)
-        reply = {"v": v, "pi_logit": pi_logit}
+        v, pi = self._forward(x, True)
```

Comment:
Funnily I had the opposite solution of changing all the "pi"s to "pi_logit".

I had the thought though: I wonder if this will create problems for loading in the old checkpoints? Since they have been saved with the keys that they were created with. Of course this can be fixed, but perhaps there should be code that fixes this before we merge the PR.

Reply:
Yeah let's go with your solution :)